fcrepo-http-api/src/test/java/org/fcrepo/integration/http/api/FedoraLdpIT.java

```diff
@@ -2476,6 +2503,41 @@ public void testContainerLastModified() throws Exception {
         }
     }
 
+    @Test
+    public void testConcurrentPutDatastreamResource() throws IOException, InterruptedException {
+        final String pid = getRandomUniqueId();
+        createObject(pid);
+
+        final String ds = "ds2";
+        final RequestThread t1 = new RequestThread(putDSFileMethod(pid, ds, TEST_FILE));
+        final RequestThread t2 = new RequestThread(putDSFileMethod(pid, ds, TEST_FILE));
+        t1.start();
+        Thread.sleep(100);
+        t2.start();
+        t1.join();
+        t2.join();
+
+        assertEquals(CREATED.getStatusCode(), t1.status());
+        assertEquals(NOT_FOUND.getStatusCode(), t2.status());
```

Comment:
Should the status code for assertion be CONFLICT.getStatusCode() instead of NOT_FOUND.getStatusCode()?